lib/rhc/commands/scp.rb

```diff
@@ -40,7 +56,7 @@ def run(_, action, local_path, remote_path)
             if sent != last_sent
               last_sent = sent
               complete = total == 0 ? 100 : ((sent.to_f/total.to_f)*100).to_i
-              $stderr.print "\r #{action}ing #{name}: #{complete}% complete. #{sent}/#{total} bytes transferred " + (sent == total ? "in #{Time.now - start_time} seconds \n" : "")
```

Comment:
Makes sense to not use `$stderr` here, but I'm not sure we can change this because of backwards compatibility (users may be relying on this for scripting, etc). Should not hurt if mentioned in the release notes though.